.circleci/config.yml

```diff
@@ -222,6 +222,9 @@ jobs:
       - run:
           command: |
             mkdir -p /tmp/logs
+            # Run the unit tests first, then the integration tests. They are separate because the integration tests
+            # require additional filtering.
```

Comment:
Additional filtering? Not sure what that means.

Reply:
Since all the tests are not tagged, if you do `go test -v -tag kubernetes ./test`, it will run all the integration tests. To run only the helm tests, I am cheating and doing `go test -v -tag helm -run TestHelm ./test`. Note that I can't do that all the time, because the unit tests aren't called that.

One way to address this is to have a separate integration test folder for k8s and helm but that might be confusing.

I am actually beginning to wonder if we should split terratest...

The real solution here, is to setup Jenkins.